nightingale_interface/screens/home_screen.py

```diff
@@ -0,0 +1,116 @@
+        button_data.parent.manager.current = "nursealertscreen"
+
+    def to_video_call(self, button_data):
+        # publish to topic to tell robot not to move
+        button_data.parent.manager.transition = SlideTransition()
+        button_data.parent.manager.transition.direction = "left"
+        cfg.LAST_SCREEN = self.home_name
+        button_data.parent.manager.current = "videocallscreen"
+        # open web browser to specific size of scree
+
+    def to_item_select(self, button_data):
+        # publish to topic to tell robot not to move
+        button_data.parent.manager.transition = SlideTransition()
+        button_data.parent.manager.transition.direction = "left"
+        cfg.LAST_SCREEN = self.home_name
+        button_data.parent.manager.current = "itemselectscreen"
+
+    def send_home(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        button_data.parent.manager.transition.direction = "left"
+        cfg.LAST_SCREEN = self.home_name
+        cfg.PENDING_ACTION = cfg.GO_HOME
+        button_data.parent.manager.current = "confirmationscreen"
+
+    def to_admin_control(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = self.home_name
+        button_data.parent.manager.current = "adminscreen"
+
+    def home_build(self):
+        screen = Screen(name=self.home_name)
+
+        # estop button
+        screen.add_widget(
+            Button(
+                background_normal="images/stop.png",
+                size_hint_x=cfg.ESTOP_XHINT,
+                size_hint_y=cfg.ESTOP_YHINT,
+                pos_hint={"center_x": cfg.ESTOP_XPOS, "center_y": cfg.ESTOP_YPOS},
+                on_release=self.estop,
+            )
+        )
+
+        screen.add_widget(
+            MDRectangleFlatButton(
+                text="Admin Control",
+                font_size=cfg.CANCEL_BUTTON_FONTSIZE,
+                pos_hint={"center_x": 0.125, "center_y": 0.9},
+                size_hint=(0.2, 0.1),
+                on_release=self.to_admin_control,
+            )
+        )
+
+        screen.add_widget(
+            MDRectangleFlatButton(
+                text="Request Items",
+                font_style="H4",
+                pos_hint={"center_x": 0.5, "center_y": 0.35},
+                size_hint=(0.4, 0.12),
+                on_release=self.to_item_select,
+            )
+        )
+        screen.add_widget(
+            MDRectangleFlatButton(
+                text="Chat with Nurse",
+                font_style="H4",
+                pos_hint={"center_x": 0.5, "center_y": 0.5},
+                size_hint=(0.4, 0.12),
+                on_release=self.to_video_call,
+            )
+        )
+
+        screen.add_widget(
+            MDRectangleFlatButton(
+                text="Urgent Assistance",
+                font_style="H4",
+                pos_hint={"center_x": 0.5, "center_y": 0.65},
+                size_hint=(0.4, 0.12),

```

Comment:
nit pick: for values that are commonly used (size hints, font styles, common allignment values) create constants so that it is easier to tweak the layout